devel/git-crypt/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            AGWA git-crypt 0.6.0
+homepage                https://www.agwa.name/projects/git-crypt/
+categories              devel
+platforms               darwin
+maintainers             @nareshov
```

Comment:
Have you considered adding `openmaintainer`? See https://guide.macports.org/chunked/reference.html#reference.keywords.maintainers for details.

Reply:
No, I hadn't. I think it makes sense - kindly go ahead and add it.